com.openwall.John.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "com.openwall.John",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "21.08",
+    "runtime-version": "22.08",
```

Comment:
23.08 is out. might as well.

Reply:
Done. I haven't tested it using 23.08 yet; but I will do it using the CI package (if there is any).